yarn-install/action.yml

```
@@ -13,7 +13,10 @@ inputs:
     default: .
     required: false
   checkout-token:
-    description: Repository checkout access token `GITHUB_TOKEN`. Required for self hosted runners
+    description: Repository checkout access token `GITHUB_TOKEN`. Required for self-hosted runners
+    required: false
+  npm-gar-token:
+    description: Repository npm Artifact Registry access token `NPM_GAR_TOKEN`. Required when using self-hosted runners with npm in GAR
```

```suggestion
    description: Repository npm Artifact Registry access token `NPM_GAR_TOKEN`. Required when using self-hosted runners with npm in GAR (Google Artifact Registry, the npm registry works as a proxy-cache, downloading and storing the public npm packages)
```

It would be also very helpful if this description would explain where to take this token 🤔

---

Done!